modules/currency-l0/src/main/scala/org/tessellation/currency/l0/Main.scala

```diff
@@ -20,8 +19,6 @@ object Main
       version = BuildInfo.version
     ) {
 
-  def dataApplication: Option[BaseDataApplicationL0Service[IO]] = None
```

Comment:
Are you sure about removing that one? It was left to be an example that one can implement a data application and provide it here

Reply:
You removed both actually. I know it defaults to None. Ok, nevermind. It should be already covered by docs I believe. Can you confirm that?